app/components/card_component.html.erb

```diff
@@ -1,9 +1,12 @@
-<div <%== attributes(classes: card_classes) %>>
-  <%= content %>
+<div class="<%== card_classes_wrapper %>">
+  <% unless content.blank? %>
+    <div <%== attributes(classes: card_classes_content) %>">
```

Comment:
If I've interpreted the code correctly, we actually want to merge the classes declared on init into the content wrapper.